source/macros.tex

```diff
@@ -156,6 +156,7 @@
 
 % Code and definitions embedded in text.
 \newcommand{\tcode}[1]{\CodeStylex{#1}}
+\newcommand{\mcode}[1]{\text{\tcode{#1}}}   % \tcode in math mode
```

Comment:
We actually have existing evidence that \tcode works in math mode as-is; see e.g. algorithms.tex:3665:
> \bigoh{N\log(N)} comparisons, where $N = \tcode{last - first}$.

\tcode is defined as \CodeStylex, which is defined as \texttt. Maybe that does work in math mode after all.

So, we could simply use `\tcode` instead of `\mcode` everywhere, knowing (through this issue) that we can differentiate between math mode and non-math-mode with `\ifmmode`, as shown above, should the need ever arise. That seems the simplest solution to me.